config/config.go

```diff
@@ -100,8 +100,8 @@ func get() (*Config, error) {
 		EnableCensusTopicFilterOption:           false,
 		EnableCensusPopulationTypesFilterOption: false,
 		EnableCensusDimensionsFilterOption:      false,
-		EnableAggregationPages:                  false,
-		EnableTopicAggregationPages:             false,
+		EnableAggregationPages:                  true,
```

Comment:
I believe these two needs to be set back to false

Reply:
rugh!

Reply:
teach me for using `-am`